hadoop-ozone/dist/src/main/compose/ozoneblockade/docker-config

```diff
@@ -32,9 +32,9 @@ OZONE-SITE.XML_ozone.scm.pipeline.owner.container.count=1
 OZONE-SITE.XML_ozone.scm.pipeline.destroy.timeout=15s
 OZONE-SITE.XML_hdds.heartbeat.interval=2s
 OZONE-SITE.XML_hdds.scm.wait.time.after.safemode.exit=30s
-OZONE-SITE.XML_hdds.scm.replication.thread.interval=5s
-OZONE-SITE.XML_hdds.scm.replication.event.timeout=7s
-OZONE-SITE.XML_dfs.ratis.server.failure.duration=25s
+OZONE-SITE.XML_hdds.scm.replication.thread.interval=6s
```

Comment:
Can you please explain the need of increasing these values ?

Reply:
In some test-cases we want to write to a pipeline where there are only two nodes in the ring (we cut the connection to the third node). In such cases SCM was destroying the pipeline immediately and the replication manager kicks-in. These values are adjusted such that the client gets enough time to write to a pipeline where there are only two healthy nodes and verify the writes.